andela/annie-hall-cfh
# 3152798642 game onboarding

* What does this PR do?
  * It enables users to be quickly and effectively onboarded onto the app

 * Description of Task to be completed?
    * clone from the github repository
    * create a branch 
    * run `yarn install` to install the dependencies
    * create an .env file to store the environmental variables
    * install `introjs` package for touring the app

 * How should this be manually tested?
   * run `yarn start` to start the server
   * navigate to `http://localhost:3000/#!/app` to play a game
   * click on the `TAKE A TOUR ` to navigate to `http://localhost:3000/#!/gametour`

 * Any background context you want to provide?
   Nil
 * What are the relevant pivotal tracker stories?
  * #152798642 - Users should be quickly and effectively onboarded onto the app
 * Screenshots (if appropriate)
<img width="1424" alt="screen shot 2017-12-14 at 11 43 07 am" src="https://user-images.githubusercontent.com/26750279/33988226-119a2596-e0c4-11e7-839e-79d10f1955f5.png">

 * Questions:
   Nil
